profile/README.md

```diff
@@ -17,23 +17,28 @@ We're constantly improving and expanding our main product — a web application
 
 Here, on GitHub we write code with ❤️. Our main stack is:
 
-- **PHP 8.3** and **Laravel 10** (always the latest versions)
-- **Vanilla JavaScript** (incl. Web Components) and **Vue.js** for Laravel Nova
-- **Server**: Docker, nginx
+- **PHP 8.3** and **Laravel 10** (we're upgrading Laravel to the latest)
```

Comment:
>(we're upgrading Laravel to the latest)

this one quickly will become outdated. I would prefer to say "The latest Laravel version"